apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/LogTo.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using Google.Cloud.Logging.V2;
+using System;
+
+namespace Google.Cloud.Diagnostics.AspNetCore
+{
+    /// <summary>
+    /// The location log entries will be logged to.
+    /// </summary>
+    public enum LogToLocation
+    {
+        /// <summary>A Google Cloud Platform project.</summary>
+        Project,
+
+        /// <summary>A Google Cloud Platform organization.</summary>
+        Organization
+    }
+
+    /// <summary>
+    /// Options for log entries will be logged, such as a project or organization.
+    /// </summary>
+    public sealed class LogTo
+    {
+        /// <summary>The location to log entries to.</summary>
+        public LogToLocation Location { get; }
+
+        /// <summary>The Google Cloud Platform project Id.</summary>
+        public string ProjectId { get; }
+
+        /// <summary>The Google Cloud Platform organization Id.</summary>
+        public string OrganizationId { get; }
+
+        internal LogTo(LogToLocation location, string projectId = null, string organizationId = null)
```

Comment:
This looks like it can be private.

Also, it might be clearer to remove these defaults and have the call sites specify all arguments, but that's a matter of taste. Up to you.